bci_tester/data.py

```diff
@@ -298,6 +301,15 @@ def create_BCI(
                     )
         else:
             available_versions = list(_DEFAULT_NONBASE_OS_VERSIONS)
+    else:
+        if available_versions:
+            for ver in available_versions:
+                if ver not in ALLOWED_BASE_OS_VERSIONS:
+                    raise ValueError(
+                        f"Invalid os version for base container: {ver}"
+                    )
+        else:
+            available_versions = list(_DEFAULT_BASE_OS_VERSIONS)
```

Comment:
This does not make a lot of sense for os & LTSS containers, as these are really only valid for a **single** os version. We will probably never hit a bug due to that, as we usually set their tags from `OS_VERSION`, but imho we could also just do this:

```Python
    else:
        if not available_versions:
            raise ValueError("OS & LTSS containers must have a version marker")
        else:
            for ver in available_versions:
                if ver not in ALLOWED_BASE_OS_VERSIONS:
                    raise ValueError(
                        f"Invalid os version for base container: {ver}"
                    )
```